docs/virtual_machines/disks_and_volumes.md

```diff
@@ -1257,3 +1257,150 @@ Example: force `writethrough` cache mode
+            name: block-disk
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 2G
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - containerDisk:
+          image: registry:5000/kubevirt/fedora-with-test-tooling-container-disk:devel
+        name: containerdisk
+      - cloudInitNoCloud:
+          userData: |-
+            #cloud-config
+            password: fedora
+            chpasswd: { expire: False }
+        name: cloudinitdisk
+      - name: block-disk
+        persistentVolumeClaim:
+          claimName: block-pvc
+---
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-block-2
+  name: vm-block-2
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/vm: vm-block-2
+    spec:
+      affinity:
+        podAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: kubevirt.io/vm
+                operator: In
+                values:
+                - vm-block-1
+            topologyKey: "kubernetes.io/hostname"
+      domain:
+        devices:
+          disks:
+          - disk:
+              bus: virtio
+            name: containerdisk
+          - disk:
+              bus: virtio
+            name: cloudinitdisk
+          - disk:
+              bus: virtio
+            shareable: true
+            name: block-disk
+        machine:
+          type: ""
+        resources:
+          requests:
+            memory: 2G
+      terminationGracePeriodSeconds: 0
+      volumes:
+      - containerDisk:
+          image: registry:5000/kubevirt/fedora-with-test-tooling-container-disk:devel
+        name: containerdisk
+      - cloudInitNoCloud:
+          userData: |-
+            #cloud-config
+            password: fedora
+            chpasswd: { expire: False }
+        name: cloudinitdisk
+      - name: block-disk
+        persistentVolumeClaim:
+          claimName: block-pvc                                        
+```
+We can try to write and read a string in the 2 guest to test the sharing.
+```bash
+$ virtctl console vm-block-1
+$ printf "Test awesome shareable disks" | sudo dd  of=/dev/vdc bs=1 count=150 conv=notrunc
+28+0 records in
+28+0 records out
+28 bytes copied, 0.0264182 s, 1.1 kB/s
+# Log into the second guest
+$ virtctl console vm-block-2
+$ sudo dd  if=/dev/vdc bs=1 count=150 conv=notrunc
+Test awesome shareable disks150+0 records in
+150+0 records out
+150 bytes copied, 0.136753 s, 1.1 kB/s
+```
+
+If you are using local devices or RWO PVCs, putting the affinity on the VMs that share the storage guarantees they will be scheduled on the same node. In the example, we put the affinity on the second VM using the label set on the first VM. If you are using shared storage with RWX PVCs, then the affinity rule is not necessary as the storage can be attached simoultaneously on multiple nodes.
```

Comment:
putting -> setting
In the example, we put the affinity -> In the example, we set the affinity
simoultaneously - simultaneously

Reply:
also change "putting the affinity on the VMs" -> "setting the affinity on the VMs"